### DISCUSSION TITLE
ha-tag: Allow monitoring any tag

### DISCUSSION THREAD

**Discussion by mathieu-mp** [zachowj/node-red-contrib-home-assistant-websocket#479] [Feature Request]
Hi,
This node is great as soon as we need a specific tag to trigger a message.

I do have a use case where I would like to trigger a message for any tag.
Would you allow tag_id to be empty and still trigger the message ?

I noticed that [devices list can already be empty](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/blob/b5d21b52ae0f45780681053bae257ccf1d33d3f7/src/controllers/Tag.js#L51), but [tag list cannot yet](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/blob/b5d21b52ae0f45780681053bae257ccf1d33d3f7/src/controllers/Tag.js#L47)

Thank you for the great work !
Regards, Mathieu.

**Comment by zachowj** [comment]
Added in v0.35.0

**Reply by mathieu-mp** [reply]
Great ! Thanks !
